Vagrantfile

```diff
@@ -14,7 +14,7 @@ end
 Vagrant.configure("2") do |config|
   # This setting will download the atlas box at
   # https://atlas.hashicorp.com/ferventcoder/boxes/win2012r2-x64-nocm
-  config.vm.box = "ferventcoder/win2012r2-x64-nocm"
+  config.vm.box = "BasicTheProgram/windows_2016"
```

Comment:
I am not sure we'd convert to another box, apologies. We would need to really review it for safety and security, as this test environment is used in *many* environments. 

And do you have vmware_fusion, hyper-v, and virtualbox versions?

Reply:
I do not have fusion, or hyper-v versions, but packer has the ability to build all of the above with a pretty simple command line evocation. 

See my comments in the google group about having git branches for each version of windows or having multiple versions of Windows supported in the VagrantFile.

I have the need for a Windows 2016 testing environment, thus these changes. I don't want to deprecated or diminish the Server 2012 R2 stuff, ideally supporting multiple test environments is the goal. Just need guidance from you on how to do that.

And, ideally, you'd take over the official config.vm.box stuff. Probably build via packer to make it easier for people to contribute to the test environment?